### ISSUE TITLE
Missing currency: MVR

### ISSUE THREAD

**Issue by KittyBloo** [RetroMusicPlayer/Paisa#402]
The Maldivian Rufiyaa (MVR) is missing from the app.

**Comment by xunafay** [comment]
Would this be correct?
```json
  {
    "code": "MVR",
    "name": "Maldivian Rufiyaa",
    "symbol": "ރ.",
    "flag": "MVR",
    "decimal_digits": 2,
    "number": 462,
    "name_plural": "Maldivian rufiyaas",
    "thousands_separator": ",",
    "decimal_separator": ".",
    "space_between_amount_and_symbol": true,
    "symbol_on_left": true,
    "pattern": "#,##,##0.00",
  },
```

**Comment by h4h13** [comment]
This for merged in #447 closing
